modules/3_helpernode/templates/helpernode_vars.yaml

```diff
@@ -54,3 +54,14 @@ ocp_installer: "${install_tarball}"
 ocp_bios: "file:///dev/null"
 ocp_initramfs: "file:///dev/null"
 ocp_install_kernel: "file:///dev/null"
+
+%{ if local_registry.enable_local_registry }
+setup_registry:
+  deploy: true
+  autosync_registry: true
+  registry_image: "${local_registry.registry_image}"
+  local_repo: "${local_registry.ocp_release_repo}"
+  product_repo: "openshift-release-dev"
+  release_name: "ocp-release"
```

Comment:
@Prajyot-Parab better have this as a variable? The way we had done this for PowerVS?

Reply:
yes, that makes sense but then in `ocp4-playbook`, we stick to this `quay.io/openshift-release-dev/ocp-release` so won't really matter. If @pravin-dsilva Can try to parameterize it in `ocp4-playbook` as well then we can make it a variable.

Reply:
Ok then it makes sense.. but you think it is good to make it a user variables please open new PRs.

Reply:
Sure we can get this PR merged and as a next step, I will focus on converting whatever remains into variables inline with ocp4-playbook changes.